specification/e-referrals-service-api.yaml

```diff
@@ -489,6 +489,9 @@ paths:
           - `REFERRING_CLINICIAN`
           - `REFERRING_CLINICIAN_ADMIN`
           - `SERVICE_PROVIDER_CLINICIAN`
+          - `SERVICE_PROVIDER_CLINICIAN_ADMIN`
+
+        If the `SERVICE_PROVIDER_CLINICIAN_ADMIN` role is used then the User ID of Service Provider Clinician that the Admin is acting on behalf of must be supplied via the `nhsd-ers-obo-user-id` header in the request.
```

Comment:
Should this start "If the `SERVICE_PROVIDER_CLINICIAN_ADMIN` role is used then the User ID of the Service..."?